src/patternfly/components/Label/label.scss

```diff
@@ -410,12 +415,15 @@
   @include pf-text-overflow;
 
   max-width: var(--pf-c-label__text--MaxWidth);
+
+  &.pf-m-truncate {
```

Comment:
Is the issue no longer to truncate labels by default?

Reply:
@mcoker This issue is still to truncate labels by default, but do so when the label runs out of room rather than at `{x}ch` width. 

![Screenshot 2023-02-22 at 3 38 23 PM](https://user-images.githubusercontent.com/5385435/220753620-de9ddb12-08ee-441e-a374-c8989bea592e.png)

Perhaps the predefined width truncation `pf-m-truncate` modifier would be implemented by simply updating `--pf-c-label__text--MaxWidth`.

Reply:
got it. Should this class be on `.pf-c-label` tho?